utilities/EluvioCli.js

```diff
@@ -265,6 +266,185 @@ const CmdAccountSignedToken = async ({ argv }) => {
   }
 };
 
+const CmdClaimerAllocate = async ({ argv }) => {
+  console.log("Claimer Allocate\n");
+  console.log("args", argv);
+
+  try {
+    let elvAccount = new ElvContracts({
```

Comment:
rename variable to elvContracts (For the other functions as well), confusing since we have an ElvAccount class